mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -3,13 +3,27 @@ AUI.add(
 	function(A) {
 		var EVENT_CLICK = 'click';
 
+		var TPL_FAILURE_MESSAGE = '<div class="alert alert-error">' +
+				'<strong>' +
+					'{warning} ' +
+				'</strong>' +
+				'{requestNotSent}' +
+			'</div>';
+
 		var TPL_FINISHED_TASK = '<li>' +
 				'{taskFinished}' +
 				'<button>' +
 					'<i class="icon-trash"></i>' +
 				'</button>' +
 			'</li>';
 
+		var TPL_SUCCESS_MESSAGE = '<div class="alert alert-success">' +
```

Comment:
Success and error message are very similar. Why not have just one template where parameter determines if it's error or success?